store/src/java/com/zimbra/cs/account/ldap/LdapProvisioning.java

```diff
@@ -1509,7 +1514,13 @@ private Account createAccount(String emailAddress, String password,
         } catch (AccountServiceException e) {
             throw e;
         } catch (ServiceException e) {
-           throw ServiceException.FAILURE("unable to create account: "+emailAddress, e);
+            // if account is not null, deleting the account details
+            // to prevent the partially created account in UI
+            if (null != acct) {
```

Comment:
Not everytime only for specific type of ServiceException

Reply:
Sorry I forgot, Done the change

output that I am getting

```
Caused by: com.zimbra.common.service.ServiceException: license error: License is not valid or Feature is in grace period due to the over usage, please disable feature to move back to normal state, features list :[SMIMEAccountsLimit/zimbraFeatureSMIMEEnabled]
```